robotiq_description/urdf/robotiq_gripper_macro.urdf.xacro

```diff
@@ -159,6 +159,24 @@
                 <geometry>
                     <mesh filename="package://robotiq_description/meshes/collision/left_finger_tip.stl" />
                 </geometry>
+                <surface>
+                    <friction>
+                    <ode>
+                        <mu1>100000.0</mu1>
+                        <mu2>100000.0</mu2>
```

Comment:
Sorry but it make me laugh when you have to set `mu` this high. Good old Gazebo!

Reply:
MORE FRICTION! MORE